constellations/gemini-user/src/internship/internship.sass

```diff
@@ -0,0 +1,56 @@
+@import "../utils/constants.sass"
+
+#view-intern
+    font-family: $fontFamilyRoboto
+    width: 65%
+    margin-left: auto
+    margin-right: auto
+
+
+    & h1
+        color: $colorTurquoise
+        margin-top: 30px
+        margin-bottom: 30px
+
+    .container-main
+        flex-direction: row
+        display: flex
+        justify-content: space-between
+        align-items: center
```

Comment:
I think CSS ordering is important because it prevents very simple mistakes and it's easy to find the properties you want if they're always consistent across all files.

I would order the following like so:
```
display
flex-direction
justify-content
align-items
```